static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -145,7 +146,9 @@ function WidgetViewerModal(props: Props) {
   return (
     <React.Fragment>
       <StyledHeader closeButton>
-        <h4>{widget.title}</h4>
+        <h4>
+          <Truncate value={widget.title} maxLength={120} />
```

Comment:
Curious if we can wrap this with `<Tooltip>` with the `showOnEllipsis` flag :eyes: